ksqldb-engine/src/main/java/io/confluent/ksql/util/BinPackedPersistentQueryMetadataImpl.java

```diff
@@ -405,7 +405,7 @@ public void onStateChange(final State newState, final State oldState) {
   @Override
   public void close() {
     loggerFactory.getLoggersWithPrefix(queryId.toString()).forEach(ProcessingLogger::close);
-    sharedKafkaStreamsRuntime.stop(queryId, false);
+    sharedKafkaStreamsRuntime.stop(queryId, true);
```

Comment:
This naming feels a bit backwards to me, which I think is how we got into this mess in the first place actually -- to me, it sounds like `stop()` means to literally stop/kill the query, whereas `close()` feels like we're just temporarily closing it to clean up resources before we "open" it back up again.

Not saying we should change this in this PR, but at the very least let's circle back later with a PR to clarify which is which in the javadocs.

Reply:
Huh that makes sense to me, we stop like we would at a stop sign, we close all the resources on close

Reply:
Fair enough -- I do think that the current way it is is the more popular/common meaning in this kind of code, it just hits a bit weird sometimes and I have to go through the code to figure out which is which. But either way docs are the answer